Source/Fuse.Models/JavaScript.Model.uno

```diff
@@ -0,0 +1,266 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Uno.Compiler;
+using Uno.IO;
+using Uno.Text;
+using Fuse;
+using Fuse.Reactive;
+using Fuse.Scripting;
+
+namespace Fuse.Models
+{
+	public class ModelJavaScript : JavaScript, IPreviewStateSaver
```

Comment:
Please name the file here ModelJavaScript.uno rather than JavaScript.Model.uno. This is no longer a partial class.

Reply:
This still applies